### PR TITLE
Create data model for time entry.

### PR DESCRIPTION
**Pull request by lydiagu**

### CODE REVIEW

File: orchestra/admin/__init__.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from orchestra.models import Step
 from orchestra.models import Task
 from orchestra.models import TaskAssignment
+from orchestra.models import TimeEntry
```
Comment (lydiagu) [inline orchestra/admin/__init__.py:8]: what is the import convention for orchestra? alphabetical?

Comment (jrbotros) [inline orchestra/admin/__init__.py:8]: Yep! We break imports into three sections based on their source: stdlib, third-party, and internal (sometimes more for more granularity, e.g., if there's a lot of `django` imports). Each section looks like this:

```
import <module>
from <module> import <lowercase_name>
from <module> import <uppercase_name>
```

File: orchestra/models.py [hunk 2]
```diff
@@ -499,6 +499,30 @@ class Status:
...
+        date (datetime.date):
+            The date the work was done.
+        time_worked (datetime.timedelta):
+            Amount of time worked.
+        assignment (orchestra.models.TaskAssignment):
+            The task assignment for which the work was done.
+        description (str): optional
+            Brief description of the work done.
+    """
+    class Meta:
```
Comment (jrbotros) [inline orchestra/models.py]: Are we supposed to include these on models? I never have

**Comment by jrbotros** [comment]
lgtm

**Pull request merged**